driver/src/order_driver.rs

```diff
@@ -49,6 +48,8 @@ pub fn run_order_listener<D, C>(
 
             let standing_order_indexes = batch_index_from_standing_orders(&standing_orders);
             let order_hash_from_contract = contract.calculate_order_hash(slot, standing_order_indexes.clone())?;
```

Comment:
I'd rename these variables to make the difference between the above `order_hash` and `contract_order_hash` more clear.

I don't really see what the difference between `contract_order_hash` and `order_hash_from_contract ` is.